hw/ip/adc_ctrl/dv/ast_adc_agent/README.md

```diff
@@ -0,0 +1,3 @@
+# AST_ADC UVM Agent
```

Comment:
Please lose the `ast_` prefix on all these classes - IMO its not needed. 

EDIT: I now realize why you added the `ast_` prefix. IMO better to lose it anyway.